apps/clinical/src/models/medicationConfig.ts

```
@@ -1,8 +1,7 @@
 import { MedicationOrdersMetadataResponse } from '@bahmni/services';
 
 export interface MedicationConfig
-  extends MedicationOrdersMetadataResponse,
-    MedicationJSONConfig {}
+  extends MedicationOrdersMetadataResponse, MedicationJSONConfig {}
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)